api/src/test/java/org/apache/iceberg/io/TestableCloseableIterable.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.iceberg.io;
+
+import java.io.IOException;
+
+public class TestableCloseableIterable implements  CloseableIterable<Integer>{
+    private Boolean closed = false;
```

Comment:
Looks like indentation (2 spaces) is off for this file (4 spaces). Can you fix it?

Reply:
Ah looks like "run checkstyle on this project" from IDEA only applies to non test files whoops!